libnavigator/proguard-rules.pro

```diff
@@ -1 +1,7 @@
 # --- Navigator ---
+
+# Refs https://github.com/mapbox/mapbox-bindgen/issues/27
+-keep class com.google.gson.** {*;}
+-keep class com.mapbox.geojson.** {*;}
```

Comment:
looks that [geo-json library](https://github.com/mapbox/mapbox-java/tree/master/services-geojson/src/main/java/com/mapbox/geojson) has a lot of `@Keep` annotations. Are you sure that this rule need here?

Reply:
Possibly not. Let's remove :+1: